CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/MapViewTopComponent.java

```diff
@@ -371,8 +382,10 @@ private void zoomLocationBasedOnGeoType(final String geoType, final String locat
         switch (geoType) {
             case GEO_TYPE_COORDINATE:
                 final String[] coordinate = location.split("[,\\s]+");
-                assert coordinate.length == 2 || coordinate.length == 3 : "Invalid coordinate syntax provided, should be comma or space separated";
-
+                if (!(coordinate.length == 2 || coordinate.length == 3)) {
```

Comment:
I would expand the boolean out. i.e. `coordinate.length != 2 && coordinate.length != 3`